src/heretic/am_map.c

```diff
@@ -155,20 +158,20 @@ static char cheat_amap[] = { 'r', 'a', 'v', 'm', 'a', 'p' };
 static byte cheatcount = 0;
 
 // [crispy] gradient table for map normal mode
-static byte antialias_normal[NUMALIAS][8] = {
+static pixel_t antialias_normal[NUMALIAS][8] = {
```

Comment:
These are strictly not pixel values, so should remain `byte` type.

Reply:
Fixed in https://github.com/fabiangreffrath/crispy-doom/pull/1111/commits/9957a41a075e35f388cd2780c7913dd9b27fe72d. Was my typo from from the time when I just started work on automap.